dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1019,3 +1019,29 @@ def test_histogram_option_integration(self):
 
         histogram, _ = num_profiler._histogram_for_profile('custom')
         self.assertEqual(100, len(histogram['bin_counts']))
+
+
+    def test_profile_merge_bin_edges_indices(self):
+        vals = [4948484949555554544949495054485054.0, 4948484948485749515554495054485054.0,
+                4948484948505251545552524952485054.0, 4948484952485048485551524952485054.0,
+                4948484948515550575556535154485054.0, 4948484950545549485651495054485054.0,
+                4948484954565649505449524950485054.0, 49484849535456545155495054485054.0,
+                4948484954515651515451495054485054.0, 4948484957575651505156554954485054.0]
+
+        data = pd.Series(vals)
```

Comment:
the profile at this level takes in a series of strings, so I think we need to do this:
```python
data = pd.Series(vals).astype(str)
```

Should fix one error, the other errors seem to be install issues that will need further debugging.